apps/condo/pages/billing/index.tsx

```diff
@@ -74,6 +77,7 @@ const AccrualsAndPaymentsPage: PageType = () => {
                 <title>{PageTitle}</title>
             </Head>
             <PageWrapper>
```

Comment:
Why not just passing it in wrapper with regex? So it' something like:
```typescript jsx
const PageWrapper = () => {
    return (
        <...>
            <GlobalHints/>
            {children}
        </...>
    )
}
```

Is this because of CC?

Reply:
I wanted to do this initially.
But it turned out that there are pages that have a `PageWrapper` component nested in each other. Because of this, duplication of tips on the pages may occur.

Since this is an experimental feature, I thought that at the moment it is better to do this. 

This can be changed later if needed.

Reply:
Got it :)